src/EnergyPlus/WindowAC.cc

```diff
@@ -892,40 +875,16 @@ namespace WindowAC {
         // METHODOLOGY EMPLOYED:
         // Obtains flow rates from the zone or system sizing arrays
 
-        auto &ZoneEqSizing(state.dataSize->ZoneEqSizing);
-
         // Using/Aliasing
         using namespace DataSizing;
-        using HVAC::CoolingCapacitySizing;
 
         // SUBROUTINE PARAMETER DEFINITIONS:
         static constexpr std::string_view RoutineName("SizeWindowAC: "); // include trailing blank space
 
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        Real64 MaxAirVolFlowDes;  // Autosized maximum air flow for reporting
-        Real64 MaxAirVolFlowUser; // Hardsized maximum air flow for reporting
-        Real64 OutAirVolFlowDes;  // Autosized outdoor air flow for reporting
-        Real64 OutAirVolFlowUser; // Hardsized outdoor ari flow for reporting
-        bool IsAutoSize;          // Indicator to autosize
-        std::string CompName;     // component name
-        std::string CompType;     // component type
-        std::string SizingString; // input field sizing description (e.g., Nominal Capacity)
-        Real64 TempSize;          // autosized value of coil input field
-        int FieldNum = 2;         // IDD numeric field number where input field description is found
-        int SizingMethod;  // Integer representation of sizing method name (e.g., CoolingAirflowSizing, HeatingAirflowSizing, CoolingCapacitySizing,
-                           // HeatingCapacitySizing, etc.)
-        bool PrintFlag;    // TRUE when sizing information is reported in the eio file
-        int zoneHVACIndex; // index of zoneHVAC equipment sizing specification
-        int SAFMethod(0);  // supply air flow rate sizing method (SupplyAirFlowRate, FlowPerFloorArea, FractionOfAutosizedCoolingAirflow,
-                           // FractionOfAutosizedHeatingAirflow ...)
-        int CapSizingMethod(0); // capacity sizing methods (HeatingDesignCapacity, CapacityPerFloorArea, FractionOfAutosizedCoolingCapacity, and
-                                // FractionOfAutosizedHeatingCapacity )
-
-        IsAutoSize = false;
-        MaxAirVolFlowDes = 0.0;
-        MaxAirVolFlowUser = 0.0;
-        OutAirVolFlowDes = 0.0;
-        OutAirVolFlowUser = 0.0;
```

Comment:
[src/EnergyPlus/WindowAC.cc:912]:(style),[variableScope],The scope of the variable 'SizingString' can be reduced.
[src/EnergyPlus/WindowAC.cc:914]:(style),[variableScope],The scope of the variable 'FieldNum' can be reduced.
[src/EnergyPlus/WindowAC.cc:915]:(style),[variableScope],The scope of the variable 'SizingMethod' can be reduced.
[src/EnergyPlus/WindowAC.cc:917]:(style),[variableScope],The scope of the variable 'PrintFlag' can be reduced.
[src/EnergyPlus/WindowAC.cc:918]:(style),[variableScope],The scope of the variable 'zoneHVACIndex' can be reduced.
[src/EnergyPlus/WindowAC.cc:919]:(style),[variableScope],The scope of the variable 'SAFMethod' can be reduced.
[src/EnergyPlus/WindowAC.cc:921]:(style),[variableScope],The scope of the variable 'CapSizingMethod' can be reduced.
[src/EnergyPlus/WindowAC.cc:924]:(style),[unreadVariable],Variable 'IsAutoSize' is assigned a value that is never used.